Files/Helpers/ContextFlyoutItemHelper.cs

```diff
@@ -447,6 +447,15 @@ public static List<ContextMenuFlyoutItemViewModel> GetBaseItemMenuItems(BaseLayo
                     IsPrimary = true,
                 },
                 new ContextMenuFlyoutItemViewModel()
+                {
+                    Text = "BaseLayoutContextFlyoutPaste/Text".GetLocalized(),
+                    Glyph = "\uE16D",
+                    Command = commandsViewModel.PasteItemsFromClipboardCommand,
+                    ShowItem = selectedItems.All(x => x.PrimaryItemAttribute == StorageItemTypes.Folder),
+                    SingleItemOnly = true,
+                    IsEnabled = App.InteractionViewModel.IsPasteEnabled && selectedItems.All(x => !x.IsRecycleBinItem),
```

Comment:
FYI, since ```ShowInRecycleBin``` isn't set to true, this item will never show in the recycle bin, so ```selectedItems.All(x => !x.IsRecycleBinItem)``` will always be false.

Reply:
thank you, i have removed the condition